invokeai/app/api/dependencies.py

```diff
@@ -66,9 +72,25 @@ def initialize(config: InvokeAIAppConfig, event_handler_id: int, logger: Logger
         logger.debug(f"Internet connectivity is {config.internet_available}")
 
         output_folder = config.output_path
+        image_files = DiskImageFileStorage(f"{output_folder}/images")
 
         db = SqliteDatabase(config, logger)
 
+        migrator = SQLiteMigrator(
+            db_path=db.database if isinstance(db.database, Path) else None,
+            conn=db.conn,
+            lock=db.lock,
+            logger=logger,
+            log_sql=config.log_sql,
+        )
+        migration_2.register_post_callback(partial(migrate_embedded_workflows, logger=logger, image_files=image_files))
```

Comment:
This post-migration callback populates the `has_workflow` column for the `images` table. It needs a logger and image file service to run.

I opted to provide those dependencies using a partial for two reasons:
1. It reduces coupling of `SQLiteMigrator` and the rest of the app.
2. It is more flexible for future migrations, which may need unique dependencies.